src/shared/parameter_v2/config/example.yaml

```diff
@@ -0,0 +1,75 @@
+# Examples of all supported Parameter types and their schemas
+--- # Metadata for this config
+metadata:
```

Comment:
Would all this have to be indented and under a parameter name? ie 

```
My_param:
  metadata:
    ....
BoolParam:
  ...
```

Reply:
No the `---` denotes a separate "document" in the YAML file. Each document can be read entirely separately. I chose to do it this way because then we can have one small set of metadata for the file, and also can avoid indenting everything.

See the section on "Multiple Documents" [here](https://rollout.io/blog/yaml-tutorial-everything-you-need-get-started/)

Reply:
Oh thats interesting, a pretty clean way of handling this

Reply:
This is now changed based on the updated proposal in the dynamic param slack channel.